src/tyr/matrix_serializer.cc

```diff
@@ -12,6 +12,24 @@ using namespace valhalla::thor;
 
 namespace {
 
+json::ArrayPtr serialize_datetime(const valhalla::Matrix& matrix,
```

Comment:
I realized there was no date_time array for a "slim" matrix. The change is contained only in this file (+ test), so I figured it's fine to cram in here